### ISSUE TITLE
IMAGINE! Interactive video workouts!

### ISSUE THREAD

**Issue by joshwouden** [ifit/Enhancements#109]
#### Enhancement Details

---

**Description**
Our customers expect interactive video workouts. Part of the reason they expect this may be because we advertise it (e.g. in our print/web ads and cool commercials: [Imagine1](https://youtu.be/C22nJGUspzM), [Imagine2](https://youtu.be/Bkrns9TKQD0), [Imagine3](https://youtu.be/kyrv3BDNg30)).

We advertise this feature because interactive video workouts are awesome!!! Let's immerse our users in awesome!

We do, of course, have the Boston Marathon, Tour de France, Atlantic Road, & Destination: Everest videos. Users love these! They want more! These videos, however, are not interactive (e.g. whether you speed up, slow down, or stop, the video continues playing at the same rate). These videos also do not follow the terrain (the treadmill does not increase the incline automatically when the video goes uphill).

IMAGINE...jogging along the Inca trail to Machu Picchu or atop Mont Blanc while your treadmill follows the terrain. Speed up, the video plays faster; or, slow down and enjoy the view! 

This is not only possible, it is completely feasible, from both a cost and development standpoint. Smaller competitors with much less resources are already doing it. Why aren't we?

[BitGym](https://www.bitgym.com/#download) is a prime example. With a monthly subscription, users can stream interactive video routes they can use on their exercise equipment. The videos are speed sensitive! You go faster, the video goes faster!

The best part of all is that we would not have to spend a fortune to produce these videos. [BitGym](https://www.bitgym.com/#download) does not make their own videos. They license them. If a crowd-funded startup can afford to license these videos, we may be able to as well.

[BitGym](https://www.bitgym.com/#download) licenses videos from [Virtual Active](http://www.vafitness.com/). Virtual Active produces professional quality first-person view videos of stunningly beautiful trails. Finding a way to speed up and slow down the videos should not be difficult (if BitGym can do it, so can we). We could even program in pre-set incline changes (matched to time-stamps on the video--something BitGym cannot do). The videos even have ambient nature sounds!

Check out some of the videos, like this one of [Costa Rica](http://www.vafitness.com/collections/hike-collection/products/costa_rica_waterfalls_hike). Tell me that this isn't what iFit should be and will be one day. That this isn't what customers expect us to be now. That this isn't what our users hope for, wish for, and dream about. That this wouldn't be a huge leap forward on our biggest selling point. 

This is doable. But not just someday--now! This is our destiny!

WE ARE IFIT!!!

**Customer Feedback**
These are just a FEW of many more comments from our users regarding video workouts:

**[More videos! I purchased iFit with the impression that I would be watching video, not still clips](https://admin.ifit.com/feedback/5300c3adfe4dd5650d003b05)**
[Where are the videos as shown on the commercial?](https://admin.ifit.com/feedback/5302cd17c4a92c660d00249d)
[Where are the high def videos for cycling that you advertise? All I see are pictures from google.](https://admin.ifit.com/feedback/530131d9f5b119800f004610)
[why i am not seeing a full color video?](https://admin.ifit.com/feedback/52cb6668c976ceb70d00080c)
[How can I get to or buy Video Workouts?](https://admin.ifit.com/feedback/5324820590b0490f280007bc)
[was very excited about ifit but disappointed in google maps, thought they would be videos](https://admin.ifit.com/feedback/540a573b16d9ce6728e043f9)
[Wished the maps would move like a video!](https://admin.ifit.com/feedback/544bfae37d5b35480c81c314)
[I thought the "street view" option would be a moving picture](http://admin.ifit.com/feedback/55826cbda35405905385f671)
[actual videos to purchase other than just the street views on the google maps?](https://admin.ifit.com/feedback/52e8372bfa460f840a000b43)
[When are you going to add more video workouts?](https://admin.ifit.com/feedback/530fe4a9460e10322b000f0c)
[Need to get video (moving pictures) on street view](https://admin.ifit.com/feedback/531c0300dd1daf6405002156)
[I'd like to down load some more videos](https://admin.ifit.com/feedback/52f2b8ceefa7b5b20f000868)
[how do I download more videos](https://admin.ifit.com/feedback/540b4b044235d97c7dfaae47)
[where are the videos of the trail/ routes on google maps??????](https://admin.ifit.com/feedback/53ab6d70e8aed4e22f0025bb)
[I am looking for scenic videos](https://admin.ifit.com/feedback/54da39a548877eea724cb2a5)
[How do I find video workouts?](https://admin.ifit.com/feedback/5313c828f89d8c3547002e15)
[Like the videos a lot better than the maps, wish you could do more :)](https://admin.ifit.com/feedback/54dd46dba35a9b762c96f2f3)
[I've been angry, too, about how he product is advertised vs. the reality of the product. I thought there was going to be more video.](https://www.ifit.com/status/554e3d35801dfa8b4985d75c)
[Only 2 video? Where the rest of them?](https://admin.ifit.com/feedback/5321f53732e517c051000440)
[where are the videos?](https://admin.ifit.com/feedback/53ef59b0bd9c1f6b29fb6bdb)
**[I would be very willing to pay for more video workouts. They're very motivating and much more enjoyable than just a map-based workout.](https://admin.ifit.com/feedback/539e1c5c272afd2a33014156)**
[Are there videos to purchase?](https://admin.ifit.com/feedback/532c889f7af643052f0011e8)
[...videos of the course, rather than just google maps?](https://www.ifit.com/status/55485dc60bce01c73cdee4ac)
[where are the video walk/runs?](https://admin.ifit.com/feedback/535177fe379d941e25004b46)
[You should ad videos of routes](https://admin.ifit.com/feedback/52eea9aa7cf060b606000e74)
[How do I get the videos with scenery to run to?](https://www.ifit.com/status/5505ed0f3b27a9196ef7bebb)
[How do I get more video's downloaded on ifit. I like them better than google maps](https://admin.ifit.com/feedback/544c70071a028c86526b8e87)
[have trouble finding video](https://admin.ifit.com/feedback/5548e0155c8315d252438391)
[does not really move with the workout](https://www.ifit.com/status/566f8ac4fffb8e2200b03600)
[the others seem to be google map workouts, which I don't like as well](https://admin.ifit.com/feedback/53b0b7dd555200f40c002a08)
[How do you get other video workouts](https://admin.ifit.com/feedback/533952cd435fcc0b180002ef)
[how about more HD video workouts?](https://admin.ifit.com/feedback/52eaa4ecf25063371e000110)
[Still waiting on the streaming video...](https://admin.ifit.com/feedback/54fcf5eecafa299f446b648c)
[The more the better, thank you!](https://admin.ifit.com/feedback/5517747865e21bec0c36421a)
[where can I find more video workout options?](https://admin.ifit.com/feedback/54899e9b5af047b47604ec62)
[How can i view more of the HD videos](https://admin.ifit.com/feedback/53385eb2d6bfb8af68001745)
[please add some video workouts!](https://admin.ifit.com/feedback/52dfb3b2494fe84a190004dd)
**[You REALLY need to do more HD videos for running! Virtual Active has it and you need it!!! At least allow Virtual Active to partner with you so we can run though the streets of Italy and not just see pictures!!](https://admin.ifit.com/feedback/54176ca6a70e3162619609e7)**

Users frequently plug BitGym right on our Community page, e.g. [If you want a THRILLING trail experience get the BitGym app.](https://www.ifit.com/status/55847c9eb96ecd6d12d148c4)

^ The demand exists!

**Images**
We frequently advertise interactive video workouts:

![image](https://cloud.githubusercontent.com/assets/7559375/11825420/258dcb88-a33c-11e5-8a1e-37608528f183.png)

![image](https://cloud.githubusercontent.com/assets/7559375/11825483/80583012-a33c-11e5-8a7e-b40348c99d6d.png)

**Notes**
@cbrammer @cwatt7 
I truly believe that there is nothing we could do that would increase the value of iFit more than this. Consumers will pay for this. It is what many think they are paying for when they subscribe. We can disappoint them, or we can wow them.

Even if we initially do not make these interactive, just having more videos in a format similar to our Destination: Everest video would be a great first step.

A stretch goal might be to eventually create a way for users to upload their own videos that were recorded while running/cycling with the Outside app. We could match the incline changes of the route recorded by the Outside app with timestamps on the video. Users could then watch videos of their own outdoor runs and races while following the same route on their treadmill/bike. We could use a feature like this to grow our video content library at a lower cost.
